tests/test_io_dict.py

```diff
@@ -40,7 +40,7 @@ def test_from_dict(data, eight_schools_params, save_warmup):
             "observed_data": eight_schools_params,
         },
         coords={
-            "school": np.arange(8),
+            "school": np.arange(8).tolist(),
```

Comment:
I am guessing the docstring of `from_dict` is not correct. There are other places where we use arrays as values in `coords` and not only it is perfectly valid code, it is actually the most common scenario

Reply:
I update the docstring for `from_dict` and now there is no need to convert to list.